templates/miq-template-ext-db.yaml

```diff
@@ -533,6 +533,24 @@ objects:
       </Location>
 
       Include "conf.d/external-auth-remote-user-conf"
+    configuration-openid-connect-auth: |
+      LoadModule auth_openidc_module modules/mod_auth_openidc.so
+
+      OIDCProviderMetadataURL      ${HTTPD_AUTH_OIDC_PROVIDER_METADATA_URL}
+      OIDCClientID                 ${HTTPD_AUTH_OIDC_CLIENT_ID}
+      OIDCClientSecret             ${HTTPD_AUTH_OIDC_CLIENT_SECRET}
+
+      OIDCRedirectURI              https://%{REQUEST_HOST}/oidc_login/redirect_uri
```

Comment:
this is problematic, REQUEST_HOST is not defined at apache config file load time. investigating alternative.